app/routes/dashboard/index.js

```diff
@@ -0,0 +1,31 @@
+import Ember from "ember";
+import AuthorizeRoute from "./../authorize";
+import AjaxPromise from "./../../utils/ajax-promise";
+
+export default AuthorizeRoute.extend({
+  renderTemplate() {
+    this.render(); // default template
+    this.render("appMenuList", {
+      into: "dashboard",
+      outlet: "appMenuList",
+      controller: "offers"
+    });
+  },
+
+  model() {
+    const recentOfferParams = {
+      state: "submitted",
+      slug: "search",
```

Comment:
`slug: "search"` --> won't that make a search for the "search" text

Reply:
`store.query('offer')` will query `/offers` endpoint. So, I have added adapter for `offer` which construct url `/offers/search` on the basis of `slug` field.

Reply:
I see ! In that case it should probably be passed as an option rather than in the data object.

According to the [docs](https://api.emberjs.com/ember-data/3.10/classes/DS.Store/methods/query?anchor=query) the function signature is `query(modelName, query, options)`, so maybe passing it in that third argument is better ?

e.g

```javascript
query("offer", recentOfferParams, { adapterOptions: { slug: 'search' } })
```

Reply:
Thanks for pointing out this. I'll add this change :) .